address.js

```diff
@@ -0,0 +1,57 @@
+const Router = require("koa-router");
```

Comment:
this file contains too many different logic parts. Split it into multiple files, for example:
- You need a `router.js` to initialize the `Router` object 
- Web3 should be in its own file
- Then in a separate file declare the route that will use the above files